docs/src/content/faq.mdx

```diff
@@ -48,27 +81,275 @@ After receiving the NFT, follow these steps to set up the Icebreaker platform:
+- Number of Icebreaker nodes currently active
+- Percentage of active nodes compared to total slots
+- Current distribution of `blockfrost-platform` versions in use
+- Icebreakers by Status: Current count of nodes in 'Up' vs 'Down' status based on the last health check
+- Status check duration: Time in milliseconds for a node to respond to the last status check ping
+- Average total response time: Mean response time over the last 10 minutes for API requests to each node
+- Incoming/Outgoing bytes: Network traffic to and from each node, indicating overall activity level
+
+This dashboard serves as the central monitoring location for all Icebreaker-related metrics and is regularly updated as nodes join or leave the network.
+
+Use these metrics to monitor your node's availability, performance and traffic levels relative to other nodes.
+Tooltips explain each metric.
+
+### What endpoints are currently supported by the Blockfrost platform?
+
+Currently, only the 'submit' endpoint is implemented in the Blockfrost platform. Additional endpoints are planned for future releases.
+
+### How can I get more detailed metrics or logs for my Icebreaker node?
+
+The `blockfrost-platform` process exposes a Prometheus metrics endpoint by default at `http://localhost:8081/metrics` with detailed operational stats.
+
+To access richer metrics:
+
+1. Ensure port `8081` is accessible
+2. Install and configure [Prometheus](https://prometheus.io/docs/introduction/overview/) to scrape the metrics endpoint
+3. Use a visualization tool like [Grafana](https://grafana.com/docs/grafana/latest/getting-started/) to graph and analyze the data.
+
+The Blockfrost team also plans to expand the public dashboard over time with additional useful metrics as the platform matures.
+
+For detailed logging, consult the `stdout` from the `blockfrost-platform` process or the `systemd` logs if running as a service.
+
+### How much bandwidth does an Icebreaker node consume?
+
+Bandwidth consumption will vary depending on overall network activity and how many requests your node serves.
+
+Rough estimates per node based on early testing:
+
+- Idle baseline: 10-50 MB/day
+- Light usage: 100-500 MB/day
+- Moderate usage: 1-5 GB/day
+- Heavy usage: 10+ GB/day
+
+Nodes also require brief bursts of higher bandwidth to submit transactions and sync with the network.
+
+Providing 5-10Mbps capacity per node should be sufficient for now, with the ability to expand as the platform scales up usage over time.
+
+Monitor your node's network metrics to gauge actual usage.
+The `Incoming / Outgoing bytes` dashboard chart is helpful for this.
+
+## Troubleshooting
+
+### What are the common setup issues with the Icebreaker platform?
+
+Common issues include:
+
+1. Incorrect configuration of the `blockfrost.toml` file
+2. Network connectivity problems, especially with IPv6
+3. Using outdated versions of the platform.
+
+### How can I resolve issues with the Blockfrost platform or installer?
```

Comment:
It would be great if we encourage people to create issues under our github project.

Reply:
@ginnun 

We do actually encourage readers to create issues on the GitHub repo, but maybe we should put greater emphasis on that. We mention it in the context of these FAQ entries: 

- How can I resolve issues with the Blockfrost platform or installer?
- How can I contribute to improving the Blockfrost platform?
- I found a typo or an error in the documentation. Where should I report it?
- What is the process for reporting a bug in the Blockfrost platform?

Let's think about how to best call more attention to encouraging people to create issues under the GitHub project. For example, the docs site doesn't yet have a "Contact us" page. We could consider adding a page like that where we call special attention to creating issues.